epochStart/metachain/systemSCs.go

```diff
@@ -313,6 +320,13 @@ func (s *systemSCProcessor) ProcessSystemSmartContract(
 		}
 	}
 
+	if s.flagInitStakingV4Enabled.IsSet() {
```

Comment:
this will be called only on the epoch when we activate the staking v4. Is this correct?
Do we need this flag to execute something only on the activation epoch?

Reply:
Yes. This is the expected behavior. We only want these actions to happen only once, hence the name StakingV4**Init**EnableEpoch. 
On this epoch, the only thing we want is to move all nodes from staking queue to auction list ("remove" waiting list).
Starting next epoch, using another flag -StakingV4EnableEpoch- we will start sorting auction nodes based on top up